### PR TITLE
Move some test files (`tets/test_xxx_utils.py`) to `tests/utils`

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

For our daily CI (and other scheduled CI) workflow, we collect the tests by using the folders under `tests`. However, there are some files under `tests` too. For some test files, they contains `TestCase` but their tests are not collected in the CI workflow run.

This PR moves `tets/test_xxx_utils.py` into `tests/utils`.

To be very sure we don't miss any test, it's probably better to come up with a solution that will collect the tests from the files under `tests` but not in any of its sub-directory. But it's more complex and might take a bit more time to deal with.

### CODE REVIEW

**Comment by SunMarc** [comment]
Yeah it's irrelevant. Let's wait until the CI is fixed !

**Review by SunMarc — APPROVED** [review]
Thanks for fixing this @ydshieh !

File: tests/utils/test_image_processing_utils.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # coding=utf-8
-# Copyright 2022 HuggingFace Inc.
+# Copyright 2023 HuggingFace Inc.
```
Comment (amyeroberts) [inline tests/utils/test_image_processing_utils.py:2]: ;)

```suggestion
# Copyright 2024 HuggingFace Inc.
```

**Review by amyeroberts — COMMENTED** [review]
Thanks for fixing!

**Review by amyeroberts — APPROVED** [review]
Thanks!

**Pull request merged**